src/config/config-window.ts

```diff
@@ -0,0 +1,3 @@
+export default interface ConfigWindow extends Window {
+    checkoutVariantIdentificationToken?: string;
```

Comment:
out of curiosity, what values can this be?

Reply:
It'd be a JWT token containing the variant ID/name.